nginx.conf.d/nginx.conf

```diff
@@ -112,6 +116,10 @@ http {
 
 		server_name excluded.##DOMAIN_NAME## excluded;
 
+		ssl_client_certificate /etc/volume/certs/cacert.pem;
```

Comment:
This will cause excluded to always ask for client certs. Don't we want a dedicated site only for client cert validation?